packages/core/src/integrations/functiontostring.ts

```diff
@@ -1,15 +1,23 @@
-import type { Integration, IntegrationClass, IntegrationFn, WrappedFunction } from '@sentry/types';
+import type { Client, Integration, IntegrationClass, IntegrationFn, WrappedFunction } from '@sentry/types';
 import { getOriginalFunction } from '@sentry/utils';
+import { getClient } from '../exports';
 import { convertIntegrationFnToClass, defineIntegration } from '../integration';
 
 let originalFunctionToString: () => void;
+let hasSetup = false;
 
 const INTEGRATION_NAME = 'FunctionToString';
 
+const SETUP_CLIENTS = new WeakMap<Client, boolean>();
+
 const _functionToStringIntegration = (() => {
   return {
     name: INTEGRATION_NAME,
     setupOnce() {
+      if (hasSetup) {
```

Comment:
do we need this? Should this not be done by setupOnce anyhow? 🤔

Reply:
ah, that's for the try-catch stuff I guess?

Reply:
> do we need this? Should this not be done by setupOnce anyhow? 🤔

Actually you're right it should be taken care of by `setupOnce`. I'll remove.